torch/csrc/generic/StorageMethods.cpp

```diff
@@ -224,11 +223,9 @@ static PyObject * THPStorage_(fromBuffer)(PyObject *_unused, PyObject *args, PyO
   auto storage = c10::make_intrusive<at::StorageImpl>(
     c10::StorageImpl::use_byte_size_t(),
     size_bytes,
-#if defined(THC_GENERIC_FILE)
-    c10::cuda::CUDACachingAllocator::get(),
-#else
+    // TODO: Should I add device kwarg to this function and use CUDA allocator,
+    // or is it alright to always use CPU?
     c10::GetDefaultCPUAllocator(),
```

Comment:
It seems like this function could not have possibly worked under CUDA...

Reply:
That's probably true. Even though this `fromBuffer` function was being generated for CUDA, it wasn't actually bound to the Python object, since there's `#if !defined(THC_GENERIC_FILE)` around the binding at the end of the file. So it was never called, I think